qwed_sdk/qwed_local.py

```
@@ -89,104 +89,213 @@ class QWED:
 
 
 
+import os
```

Ruff (0.15.1)

[error] Redefinition of unused `os` from line 34: `os` redefined here

Remove definition: `os`

(F811)